2-projects/1-todo-list/script.js

```diff
@@ -17,9 +61,27 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
```

Comment:
> and remember to reset the input field to be blank after creating a todo!

This part has been skipped and also it would be good to implement logic that prevents from adding duplicate tasks or tasks that are empty strings